### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by DanielFran**
-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by DanielFran** [comment]
@pascalgrimaud can you merge this before today's release? Should be OK

**Comment by DanielFran** [comment]
@pascalgrimaud @jdubois Do not forget that is necessary a release of jhipster bom dependencies

File: generators/generator-constants.js [hunk 1]
```diff
@@ -27,29 +27,29 @@ const DOCKER_MONGODB = 'mongo:4.0.12';
...
 const DOCKER_JHIPSTER_CONSOLE = 'jhipster/jhipster-console:v4.1.0';
 const DOCKER_JHIPSTER_CURATOR = 'jhipster/jhipster-curator:v4.1.0';
 const DOCKER_JHIPSTER_ELASTICSEARCH = 'jhipster/jhipster-elasticsearch:v4.1.0';
 const DOCKER_JHIPSTER_LOGSTASH = 'jhipster/jhipster-logstash:v4.1.0';
 const DOCKER_JHIPSTER_IMPORT_DASHBOARDS = 'jhipster/jhipster-import-dashboards:v4.1.0';
 const DOCKER_JHIPSTER_ZIPKIN = 'jhipster/jhipster-zipkin:v4.1.0';
-const DOCKER_TRAEFIK = 'traefik:1.7.14';
-const DOCKER_CONSUL = 'consul:1.6.0';
+const DOCKER_TRAEFIK = 'traefik:1.7.15';
+const DOCKER_CONSUL = 'consul:1.6.1';
```
Comment (pascalgrimaud) [inline generators/generator-constants.js]: there is a problem with consul:

```
➜ docker pull consul:1.6.1
1.6.1: Pulling from library/consul
no matching manifest for linux/amd64 in the manifest list entries
```

Comment (DanielFran) [reply generators/generator-constants.js]: Yes I saw it now, but it is available in dockerhub !?!

Comment (DanielFran) [reply generators/generator-constants.js]: I will roolback it now...

Comment (pascalgrimaud) [reply generators/generator-constants.js]: available at Docker Hub but it seems broken: https://hub.docker.com/_/consul

Comment (DanielFran) [reply generators/generator-constants.js]: They created it again...
I will include it again to see if now it is OK

Comment (DanielFran) [reply generators/generator-constants.js]: @pascalgrimaud This seems OK now!

**Comment by DanielFran** [comment]
@pascalgrimaud @jdubois Can I merge it?

**Comment by pascalgrimaud** [comment]
If CI passed, go !

**Comment by DanielFran** [comment]
@pascalgrimaud Just waiting Travis...

**Pull request merged**